jib-core/src/main/java/com/google/cloud/tools/jib/filesystem/PermissionsHelper.java

```diff
@@ -18,8 +18,8 @@
 
 import com.google.common.base.Preconditions;
 import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
 import java.nio.file.attribute.PosixFilePermission;
-import java.util.HashSet;
 import java.util.Map.Entry;
 import java.util.Set;
 
```

Comment:
nit: `permissionMap` to include a short javadoc like `Maps from {@link PosixFilePermission} to their intended file permissions bit.`